### PR TITLE
Dane/install/snvstory

### PR DESCRIPTION
**Pull request by kennedydane**
Mainly this was to install snvstory.

But I did a bunch of refactoring too — just a reorganising of the container things.

### CODE REVIEW

**Comment by Overlytic** [comment]
nice! i like the sorting into separate bio and astro folders for the container recipes!

**Review by Overlytic — APPROVED** [review]

File: ansible/roles/compiled/tasks/common/main.yml [hunk 1]
```diff
@@ -1125,3 +1125,10 @@
       - intel
       - inteloneapi
       - intelmkl
+
+- name: Install Micromamba
```
Comment (Overlytic) [inline ansible/roles/compiled/tasks/common/main.yml:1129]: Ooh i don't know micromamba! _googles_ very interesting.

File: ansible/roles/compiled/templates/common/micromamba.lua [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+local mamba_dir = "{{ install_dir }}"
+local funcs = "__mamba_exe __mamba_hashr __mamba_xctivate micromamba __add_sys_prefix_to_path"
+local the_shell = os.getenv("SHELL")
+
+local home = os.getenv("HOME")
+
+
+-- Initialize mamba
+execute{cmd="source " .. mamba_dir .. "/shell_hook", modeA={"load"}}
```
Comment (Overlytic) [inline ansible/roles/compiled/templates/common/micromamba.lua:23]: This is a really interesting lua file. I'm wondering why funcs is declared and never used.

Comment (Overlytic) [reply ansible/roles/compiled/templates/common/micromamba.lua:23]: oh i guess its an environmental variable that is used by the shell_hook script

Comment (Overlytic) [reply ansible/roles/compiled/templates/common/micromamba.lua:23]: very cool :)

Comment (kennedydane) [reply ansible/roles/compiled/templates/common/micromamba.lua:23]: I got that from the anaconda lua file, which I got from elsewhere? (It's referenced in the anaconda lua file if you're interested) Have

**Pull request merged**